db/migrate/20240724070523_rename_column_name_in_table_name.rb

```diff
@@ -0,0 +1,5 @@
+class RenameColumnNameInTableName < ActiveRecord::Migration[7.1]
```

Comment:
What is this name of Migration ?
Please don't use google or GPT blindly

Reply:
RenameHr__idInTableNameCandidate

Reply:
RenameHr_idinCandidate

Reply:
user_id is store hr_id in candidate table based on role hr